src/js/actions/transform.js

```diff
@@ -231,7 +235,40 @@ define(function (require, exports) {
         // Used to calculate the new top/left positions of all layers in relation to top/left
         // of the group
         var overallWidthRatio = size.w ? size.w / overallBounds.width : 0,
-            overallHeightRatio = size.h ? size.h / overallBounds.height : 0;
+            overallHeightRatio = size.h ? size.h / overallBounds.height : 0,
+            overallLeftOffset = 0,
+            overallTopOffset = 0,
+            direction;
+
+        if (size.hasOwnProperty("w")) {
+            direction = refPoint.charAt(0);
+            switch (direction) {
+                case "l":
```

Comment:
I don't think you really need to specify this case, since there is not default case.  Unless you left it in for "self documentation" sake, then carry on